### PR TITLE
Remove gui dependencies

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: maps_printer.py [hunk 1]
```diff
@@ -98,21 +94,27 @@ def initGui(self):
...
                               self.iface.mainWindow()
                               )
         self.helpAction = QAction(GuiUtils.get_icon('about.png'),
                                   self.tr(u'Help'), self.iface.mainWindow()
                                   )
 
         # Connect the action to the run method
-        self.action.triggered.connect(self.run)
+        #self.exportProject.triggered.connect(MapsPrinterProvider.algorithm(ExportLayoutsFromProject))
+        self.exportProject.triggered.connect(self.run)
```
Comment (DelazJ) [inline maps_printer.py]: The idea here is to add shortcuts for algs and clicking the button will open the corresponding dialog

File: maps_printer.py [hunk 2]
```diff
@@ -123,9 +125,9 @@ def unload(self):
         """Removes the plugin menu item and icon from QGIS GUI."""
 
         QgsApplication.processingRegistry().removeProvider(self.provider)
-        self.iface.removePluginMenu(u'&Maps Printer', self.action)
+        self.iface.removePluginMenu(u'&Maps Printer', self.exportProject)
+        self.iface.removePluginMenu(u'&Maps Printer', self.exportFolder)
         self.iface.removePluginMenu(u'&Maps Printer', self.helpAction)
-        self.iface.removeToolBarIcon(self.action)
 
     def run(self):
```
Comment (DelazJ) [inline maps_printer.py]: No idea if this is needed or how we could transform it to open the corresponding algorithm dialog

**Review by DelazJ — COMMENTED** [review]
Hi @gacarrillor 
May I request your feedback on this please? thanks

File: maps_printer.py (hunk 1 above)
Comment (gacarrillor) [reply maps_printer.py]: So, something like https://gis.stackexchange.com/questions/168562/assigning-shortcut-to-pyqgis-script, right? I think such answer worked for QGIS 2.x.

Do you have already any shortcuts in mind?
Do you want me to have a look at it and implement it for both alfgorithms?

Comment (DelazJ) [reply maps_printer.py]: OK. I improved a bit what I was looking for based on discussion on dev ML. Not a keyboard shortcut but a menu shortcut, same as menu "Vector -->Geometry tools --> Centroids".
Issue with my last change is that I don't know how I could make the openDialog method open the corresponding algorithm depending on the trigger menu (ie "exportProject" launches "mapsprinter:ExportLayoutFromProject" and "exportFolder" launches "mapsprinter:ExportLayoutFromFolder" using the single method)

Comment (DelazJ) [reply maps_printer.py]: Looks like I got it.

**Pull request merged**